cli/src/cli.rs

```diff
@@ -116,6 +116,13 @@ pub struct RunCmd {
 	/// commonly `127.0.0.1:6831`.
 	#[clap(long)]
 	pub jaeger_agent: Option<std::net::SocketAddr>,
+
+	/// Add the destination address to the `pyroscope` agent.
+	///
+	/// Must be valid socket address, of format `IP:Port`
+	/// commonly `127.0.0.1:4040`.
+	#[clap(long)]
+	pub pyroscope_agent: Option<std::net::SocketAddr>,
```

Comment:
It would be nice to support hostname resolution here and not just IP addresses.

Reply:
Technically there is no limitation

Reply:
When I set `--pyroscope-agent=pyroscope.pyroscope:4040`, I get the following error: `error: Invalid value for '--pyroscope-agent <PYROSCOPE_AGENT>': invalid IP address syntax`. I think to support hostnames, the parameter needs to be a string and resolved using [ToSocketAddrs](https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html).